### DISCUSSION TITLE
برای سایت های وردپرسی

### DISCUSSION THREAD

**Discussion by alirezarah** [tapsi-pack/api-doc#20] [Ideas]
با سلام و وقت بخیر 
آیا میشه این api رو برای سایت های وردپرسی هم استفاده کرد اگر میشه لطفا راهنمای نصب ویا اجرراش رو ارسال کنید با تشکر

**Comment by TRahnovin** [comment]
> با سلام و وقت بخیر آیا میشه این api رو برای سایت های وردپرسی هم استفاده کرد اگر میشه لطفا راهنمای نصب ویا اجرراش رو ارسال کنید با تشکر

سلام افزونه حمل و نقل مسیر را در گوگل جستجو کنید
